sw/device/sca/ecc256_keygen_serial.c

```diff
@@ -144,6 +198,71 @@ static void p256_run_keygen(uint32_t mode, const uint32_t *seed,
   sca_set_trigger_low();
 }
 
+static void ecc256_ecdsa_secret_keygen_batch(const uint8_t *data,
+                                             size_t data_len) {
+  uint32_t num_traces = 0;
+  uint32_t out[kEcc256SeedNumWords];
+  uint32_t batch_digest[kEcc256SeedNumWords];
+  uint8_t dummy[kEcc256SeedNumBytes];
```

Comment:
Also, would you mind declaring these variables closest to where they're first used to comply with the style guide, rather than at the beginning of the function?

Reply:
In some cases you can also inline the variable declaration (following the style rule @jadephilipoom mentioned):

```c
uint32_t num_traces = read_32(data);
```

Reply:
Same as in the comments above, I stuck to the other batch mode implementations where it is done similar. 
But it's a legitimate objection which should be corrected for all files.